### PR TITLE
Update copyrights for externally sourced code

### PR DESCRIPTION
**Pull request by massdosage**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -266,15 +266,7 @@
           <configuration>
             <!-- excluding files that don't need a header update -->
             <excludes>
-              <exclude>src/main/java/com/hotels/bdp/waggledance/api/validation/constraint/TunnelRoute.java</exclude>
```
Comment (massdosage) [inline pom.xml:269]: These classes are no longer in this project so they can be removed from this exclusion section.

File: waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/MetaStoreProxyServer.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (C) 2015-2017 The Apache Software Foundation and Expedia Inc.
```
Comment (massdosage) [inline waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/MetaStoreProxyServer.java:2]: I've added the missing comma(!) and updated the year.

Comment (JayGreeeen) [reply waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/MetaStoreProxyServer.java:2]: out of interest, will this ever be updated to Expedia Group?

Comment (massdosage) [reply waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/MetaStoreProxyServer.java:2]: According to our lawyers, in the near future, no. I'm guessing it's the name of some legally registered entity which is hard/expensive to change.

**Review by nvitucci — APPROVED** [review]

File: waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/MetaStoreProxyServer.java (hunk 2 above)
Comment (JayGreeeen) [reply waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/MetaStoreProxyServer.java:2]: ahh okay

**Review by JayGreeeen — APPROVED** [review]

**Pull request merged**